Description: Use result of 'write' function.
 Fix GCC warning about "ignoring return value of 'write', declared with
 attribute warn_unused_result".
 .
 This warning was observed on:
   * Maverick with GCC 4.4.3 (Ubuntu 4.4.3-4ubuntu5)
   * Natty with GCC 4.5.2 (Ubuntu/Linaro 4.5.2-2ubuntu1)
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: no
Last-Update: 2011-01-31

Index: uwsgi-0.9.6.4/plugins/rack/rack_plugin.c
===================================================================
--- uwsgi-0.9.6.4.orig/plugins/rack/rack_plugin.c	2010-09-27 05:51:53.000000000 +0300
+++ uwsgi-0.9.6.4/plugins/rack/rack_plugin.c	2010-09-27 05:53:01.000000000 +0300
@@ -250,6 +250,7 @@
 
 	int error;
 	int i;
+	ssize_t wsize;
 
 	struct http_status_codes *http_sc;
 
@@ -345,7 +346,7 @@
 
 		uwsgi_log("wsgi_req ptr: %p\n", wsgi_req);
 
-		write(wsgi_req->poll.fd, "\r\n", 2);
+		wsize = write(wsgi_req->poll.fd, "\r\n", 2);
 
 		uwsgi_log("PTR: %p\n", RARRAY(ret)->ptr[2]);
 
